docker/nmslib/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM amazonlinux:latest
+WORKDIR /app
```

Comment:
Forgot to remove line 2 from a previous config. However, lines 4,5,7,8 still need to be in that order since each of those layers depends on the previous layer. `useradd` is a part of `util-linux` and the `test` user isn't in the `/etc/passwd` entries until `useradd` is ran, etc. Lines 7/8 can be switched with each other, though I think that change is trivial.